config/routes.rb

```diff
@@ -1,15 +1,18 @@
 Rails.application.routes.draw do
   get 'cart_items/create'
 
-   root 'welcome#index'
+  root 'welcome#index'
 
-   post '/login', to: 'sessions#create'
+  post '/login', to: 'sessions#create'
 
-   delete '/logout', to: 'sessions#destroy', as: 'logout'
-  # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
+  delete '/logout', to: 'sessions#destroy', as: 'logout'
+
+  get '/cart', to: "sessions#showcart", as: "showcart"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.